frontend/src/components/TimeRangeSelect.vue

```diff
@@ -1,58 +1,80 @@
 <template>
-    <v-select outlined
-              hide-details
-              dense
-              label="Timerange"
-              item-value="value"
-              item-text="label"
-              :items="options"
-              clearable
-              v-bind="$attrs"
-              v-on="$listeners"
+    <v-select
+      outlined
+      hide-details
+      dense
+      label="Timerange"
+      item-value="value"
+      item-text="label"
+      :items="options"
+      clearable
+      v-bind="$attrs"
+      v-on="$listeners"
     />
 </template>
 
 <script lang="ts">
 import Vue from 'vue'
 
+interface BuilderConfig {
```

Comment:
I think you can improve this by changing it to:

```ts
interface BuilderConfig {
  [key in BuilderConfigKeys]: number[];
}
```

if so you should be able to remove the `as BuilderConfigKeys[]` in your method. I know this works for enums, not sure if also by string literals. I'm using enums more often.

Reply:
I tried as you suggested but I got this error:
```
A computed property name in an interface must refer to an expression whose type is a literal type or a 'unique symbol' type.
A computed property name must be of type 'string', 'number', 'symbol', or 'any'.
```